l10n_it_fatturapa_out_welfare/views/account_invoice_views.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!--
+  ~ Copyright 2022 Simone Rubino - TAKOBI
+  ~ License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+  -->
+
+<odoo>
+    <!-- Views -->
+    <record id="invoice_form" model="ir.ui.view">
+        <field name="name">Add Welfare Fund Type Amount to Invoice Form View</field>
+        <field name="model">account.invoice</field>
+        <field name="inherit_id" ref="account.invoice_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='invoice_line_ids']/tree/field[@name='price_subtotal']" position="before">
+                <field name="welfare_grouping_fund_type_amount_id" invisible="True"/>
+                <field
+                    name="welfare_fund_type_amount_ids"
+                    widget="many2many_tags"
+                    attrs="{
+                        'readonly': [
+                            ('welfare_grouping_fund_type_amount_id', '!=', False),
+                        ],
+                    }"
+                />
+            </xpath>
+            <notebook position="after">
+                <field name="should_regenerate_welfare_lines" invisible="True"/>
+                <button
+                    name="button_regenerate_welfare_lines"
+                    type="object"
+                    string="Regenerate Welfare Lines"
```

Comment:
Penso sia meglio "Generate"

Reply:
Grazie della review!
Ho messo "Regenerate" perché le righe esistenti vengono eliminate, e poi vengono create nuove righe.
Con "Regenerate" dovrebbe essere chiaro all'utente che le modifiche fatte alle righe generate andranno perse.

Reply:
Lo scenario tipico consiste nell'inserire la cassa sulle righe e poi generare la nuova riga. Quindi "Rigenera" in questo caso è fuorviante.
Poi se l'utente clicca nuovamente "genera" e la riga di cassa viene sovrascritta, è un comportamento intuitivo del "genera"; non c'è quindi bisogno di chiamarlo "rigenera" per un caso secondario che sarebbe comunque chiaro.

Reply:
In effetti la prima volta (caso d'uso più comune) "Rigenera" ha poco senso, ho modificato in "Genera".
Ho aggiunto però una nota nel `help` riguardo l'eliminazione delle righe esistenti perché secondo me non è così intuitivo che saranno eliminate: da un pulsante con scritto "Genera" mi aspetterei solo la creazione di nuovi oggetti.
Puoi controllare se ora va meglio?